src/display.js

```diff
@@ -0,0 +1,42 @@
+class Display {
+  display = () => {
+    /* tasks list */
+    const tasks = [
+      {
+        index: 1,
+        description: 'structure todo list',
+        completed: 'false',
```

Comment:
- [ ] It would be better to make the "false" a  boolean value. This can be done by removing the quotations that are surrounding it.